src/pages/Inventory.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { useState } from "react";
+import "../stylesheets/Inventory.css";
+
+// used for temp unique item ids
+const generateRandomId = () => {
+  return Math.floor(Math.random() * 1000000);
+};
+
+export default function InventoryManagement() {
+  const [items, setItems] = useState<Item[]>([]);
+  const [selectedItem, setSelectedItem] = useState<Item | null>(null);
+  const [itemName, setItemName] = useState<string>("");
+  const [itemAbout, setItemAbout] = useState<string>("");
+  const [itemPrice, setItemPrice] = useState<number>(0.0);
+  const [itemQuantity, setItemQuantity] = useState<number>(0);
+  const [itemId, setItemId] = useState<number | null>(null);
+  
+
+  const handleInputNameChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setItemName(event.target.value);
+  };
+
+  const handleInputAboutChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setItemAbout(event.target.value);
+  };
+
+  const handleAddItem = async () => {
+    if (itemName.trim() !== "") {
+      try {
+        const newItem = {
+          name: itemName,
+          id: generateRandomId(),
```

Comment:
Do we want to remove the ID thing for adding items?  That should be handled by the backend I think -- I believe the database auto-increments the item ID.